packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -101,6 +102,10 @@ const AppContent = (): JSX.Element => {
     return { imageSrc: HeaderArbitrumLogoMainnet, className }
   }, [l1.network])
 
+  if (isRinkeby || isArbitrumRinkeby) {
+    return <Notifications />
+  }
+
```

Comment:
These changes aren't needed, as this logic is handled inside the `Notifications` component